src/Adapters/EventSource.Net46/Implementation/EventSourceListenerEventSource.cs

```diff
@@ -0,0 +1,33 @@
+﻿//-----------------------------------------------------------------------
+// <copyright file="EventSourceListenerEventSource.cs" company="Microsoft">
+//     Copyright (c) Microsoft Corporation. All rights reserved.
+// </copyright>
+//-----------------------------------------------------------------------
+
+namespace Microsoft.ApplicationInsights.EventSourceListener.Implementation
+{
+    using System.Diagnostics.Tracing;
+
+    /// <summary>
+    /// EventSource for reporting errors and warnings from the EventSourceListener telemetry module.
+    /// </summary>
+    [EventSource(Name = ProviderName)]
```

Comment:
I hear you, but I am actually referring to this name from EventSourceTelemetryModule.EnableAsNecessary() method (to avoid tracking events from our own diagnostics event source).